### PR TITLE
Wired up display sums menu item.

### PR DESCRIPTION
**Pull request by manwithtwowatches**
Put modal inits into an options dropdown directive. Not sure if that's the best thing to do but it felt right at the time.

Once the dropdown has been styled then we'll need an ng-class to show/hide the tick.

### CODE REVIEW

**Comment by manwithtwowatches** [comment]
https://www.openproject.org/work_packages/7590

File: app/assets/javascripts/angular/directives/work_packages/options-dropdown-directive.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            });
+        }
+      };
+
+      scope.toggleDisplaySums = function(){
+        scope.query.displaySums = !scope.query.displaySums;
+      }
+    }
+  };
+}]);
```
Comment (tilt) [inline app/assets/javascripts/angular/directives/work_packages/options-dropdown-directive.js:69]: :+1: for moving this into its own context

**Comment by myabc** [comment]
I meant to comment on this via email, and it didn't get sent…

My original code was neither elegant nor DRY, I'm not such a fan of this approach. I can foresee the modals needing to be invoked from elsewhere in the work packages UI, not only from these toolbar dropdowns.

**Comment by tilt** [comment]
The "remote controls" for these modals can still be required via DI can't they?

**Pull request merged**